index.d.ts

```diff
@@ -20,6 +21,7 @@ declare module 'hardhat/types/config' {
       strict: boolean
       only: string[],
       except: string[],
+      outputFile: string
```

Comment:
Also need to set a default value in `index.js`.

Reply:
What would you recommend the default value be? 
- `'./contractSizerOutput.log'`
- `null`
- `''`

Reply:
Probably `null`.